superset/databases/api.py

```diff
@@ -838,6 +848,70 @@ def validate_sql(self, pk: int) -> FlaskResponse:
         except DatabaseNotFoundError:
             return self.response_404()
 
+    @expose("/oauth2/", methods=["GET"])
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}.oauth",
+        log_to_statsd=False,
+    )
```

Comment:
qq, is there a benefit to having the separate decorator for statsd rather than including it in the event logger?

Reply:
No, let me fix it.